src/sources/video_card.test.ts

```diff
@@ -0,0 +1,36 @@
+import getVideoCard from './video_card'
+
+describe('Sources', () => {
+  describe('videoCard', () => {
+    const videoCard = getVideoCard()
+    it('to be defined', () => {
+      expect(videoCard).toBeDefined()
+    })
+
+    // Doesn't work
+    // it needs the object to be exactly like what I provided and I only need the keys
```

Comment:
If you only need the keys, you can do:

```js
expect(Object.keys(videoCard)).toEqual(['vendor', 'renderer'])
```

It's very strange that it throws an error. I've tested it locally, it works. What error do you get? Maybe you get an undefined `videoCard` value in Internet Explorer.

But I recommend making a test that checks both the keys and the the values:

```js
expect(videoCard).toEqual({
  vendor: jasmine.any(String),
  renderer: jasmine.any(String),
})
```